src/gflownet/data/sampling_iterator.py

```diff
@@ -28,7 +29,7 @@ def __init__(
         self,
         dataset: Dataset,
         model: nn.Module,
-        batch_size: int,
+        cfg: Config,
         ctx,
         algo,
         task,
```

Comment:
Is there a reason why some arguments are parameterized by the config, while others (ratio, stream, etc) need to be provided to the class ?

Reply:
doc string was not updated.

Reply:
Good point; user may want to instantiate multiple `SamplingIterator` instances with different batch_sizes, rolled back.